pkg/tsdb/cloudwatch/models/cloudwatch_query_test.go

```diff
@@ -294,7 +296,7 @@ func TestQueryJSON(t *testing.T) {
 	var res metricsDataQuery
 	err := json.Unmarshal(jsonString, &res)
 	require.NoError(t, err)
-	assert.Equal(t, "timeSeriesQuery", res.QueryType)
+	assert.Equal(t, "timeSeriesQuery", res.Type)
```

Comment:
Missed this, but this test could have been deleted when the type reading behavior was changed.